docs/contributing/plugin_api/index.rst

```diff
@@ -1,6 +1,11 @@
 Plugin API
 ==========
 
+The Pulp Plugin API is versioned separately from Pulp Core. It is governed by `semantic
```

Comment:
I think the user would see the latest version in the release notes. I will add a link for easier access